conductr_cli/sandbox_stop_jvm.py

```diff
@@ -6,86 +15,71 @@ def stop(args):
 
     :param args: args parsed from the input arguments
     """
-
-    core_pids = find_conductr_core_pids(args.image_dir)
-    core_killed_pids, core_hung_pids = kill_processes(core_pids)
-
-    agent_pids = find_conductr_agent_pids(args.image_dir)
-    agent_killed_pids, agent_hung_pids = kill_processes(agent_pids)
-
-    if core_hung_pids or agent_hung_pids:
-        if core_hung_pids:
-            log_hung_core_processes(core_hung_pids)
-
-        if agent_hung_pids:
-            log_hung_agent_processes(agent_hung_pids)
-
-        raise_hung_processes_error(core_hung_pids, agent_hung_pids)
-    else:
-        log_terminated_successfully(core_killed_pids, agent_killed_pids)
-
-
-def find_conductr_core_pids(run_dir):
-    """
-    Finds the PIDs of the ConductR core from the output of the ps process, looking for java process which is running
-    of the sandbox image.
-    :param run_dir: directory of where ConductR core is running from.
-    :return: the list of the ConductR core pids.
-    """
-    pass
-
-
-def find_conductr_agent_pids(run_dir):
-    """
-    Finds the PIDs of the ConductR agent from the output of the ps process, looking for java process which is running
-    of the sandbox image.
-    :param run_dir: directory of where ConductR agent is running from.
-    :return: the list of the ConductR agent pids.
-    """
-    pass
-
-
-def kill_processes(pids):
+    log = logging.getLogger(__name__)
+    core_info, agent_info = sandbox_common.resolve_conductr_binrary_info(args.image_dir, args.image_version)
```

Comment:
Renamed it to `resolve_conductr_info` because now it does not contain only binary related information.